build_carddb_lorcana.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+"""
+This script downloads the latest Disney Lorcana card data from https://lorcanajson.org/ and processes
+it to turn the highly-structured data there into a flat list of card names to descriptions
+formatted to send down the chat.
+"""
+
+import common
+common.FRAMEWORK_ONLY = True
+
+import json
+import sys
+import zipfile
+
+import psycopg2
+
+from common.card import CARD_GAME_LORCANA
+from common.config import config
+import build_carddb_mtg
+
+def main():
+	forceRun = '-f' in sys.argv
+
+	was_downloaded = build_carddb_mtg.do_download_file("https://lorcanajson.org/files/current/en/allCards.json.zip", "LorcanaCards.json.zip")
+	if not was_downloaded and not forceRun:
+		print("No Lorcana card update available, stopping update")
+		return
+
+	print("Loading card data")
+	with zipfile.ZipFile(r"LorcanaCards.json.zip") as card_zip_file:
+		with card_zip_file.open("allCards.json") as cardFile:
+			carddata = json.load(cardFile)
+
+	print("Processing and storing card data")
+	with psycopg2.connect(config['postgres']) as conn, conn.cursor() as cur:
+		cur.execute("DELETE FROM cards WHERE game = %s", (CARD_GAME_LORCANA, ))
+		for card in carddata['cards']:
+			cur.execute("INSERT INTO cards (game, filteredname, name, text, hidden) VALUES (%s, %s, %s, %s, %s)", (
+				CARD_GAME_LORCANA,
+				card['simpleName'],
```

Comment:
Using `card['simpleName']` for `filteredname` makes some cards like Ursula's Shell Necklace unsearchable by full name because `card['simpleName']` leaves apostrophes alone but `clean_text()` strips them. The spaces would probably also lead to differences between card commands.

```suggestion
				clean_text(card['fullName']),
```

Reply:
I didn't realise clean_text treated apostrophes differently, good point.
Lorcana has cards for the character 'Te Kā' from Moana, and that 'ā' isn't cleaned yet, so I've added that too